test/init.js

```diff
@@ -9,13 +9,22 @@ module.exports = require('should');
 
 var DataSource = require('loopback-datasource-juggler').DataSource;
 
+/** these are the env variables in jenkins **/
+if (process.env.CI && (process.env.BUILD_NUMBER || process.env.BUILD_ID) &&
+  (process.env.nodeVersion || process.env.node)) {
+  var buildNumber = process.env.BUILD_NUMBER || process.env.BUILD_ID;
+  var nodeVersion = process.env.nodeVersion || process.env.node;
+  var os = process.env.OS || 'LINUX';
```

Comment:
Nope. These are jenkins specific env variables and `platform` is not an env variable in jenkins.